client/src/views/Game.vue

```diff
@@ -58,6 +58,9 @@
           <!-- Game stats -->
           <v-row>
             <v-card class="ma-3 pa-6" outlined tile>
+              <p>
+              <v-switch v-model="switch1" :label="`Minimize Players`"></v-switch>
```

Comment:
hmmm.. on my thoughts. I think the label probably would be good to emphasize you are locking the pane.. Like `Lock Players Menu`. But that's just me.